lib/metrics/amplitude.js

```diff
@@ -106,14 +106,14 @@ const EVENT_PROPERTIES = {
 }
 
 module.exports = (log, config) => {
-  if (! log || ! Array.isArray(config.oauth.clientIds)) {
-    throw new TypeError('Bad argument')
+  if (! log || ! config.oauth.clientIds) {
```

Comment:
Can `clientIds` be null when default specified in config? Doesn't hurt either way.